litellm/utils.py

```diff
@@ -729,12 +730,15 @@ def function_setup(  # noqa: PLR0915
                 and isinstance(messages[0], dict)
                 and "content" in messages[0]
             ):
+
```

Comment:
hmm i don't think we even need to run pre_call_rules by default. Most user don't use this

Reply:
instead can we just check len(litellm.pre_call_rules) > 0 

use a helper static method in rules.py for this